src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -521,7 +521,10 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         tag,
         namePrefix,
     }: IFeatureProjectUserParams): Promise<IFeatureOverview[]> {
-        let query = this.db('features').where({ project: projectId });
+        let query = this.db('features');
+        if (projectId) {
```

Comment:
this is only used by one existing endpoint. A breaking change is that now if projectId is empty we return all features.